user-interface/src/main/resources/application.properties

```diff
@@ -162,3 +162,7 @@ qbic.communication.documentation.url=${DOCUMENTATION_URL:https://qbicsoftware.gi
 qbic.communication.contact.email=${CONTACT_EMAIL:support@qbic.zendesk.com}
 qbic.communication.contact.subject=${CONTACT_SUBJECT:Data Management Question}
 qbic.communication.api.url=${API_URL:https://download.qbic.uni-tuebingen.de/swagger-ui.html}
+################# Terminology Service ########################################
+tib.terminology.service.endpoint.select=${TIB_SERVICE_API_SELECT_ENDPOINT:/ts4tib/api/select}
```

Comment:
NIT: I suggest renaming to `terminology.service.tib.*` instead so other terminology services might be added in the future.